Autopilot/boardfiles/Src/freertos.c

```diff
@@ -92,26 +93,28 @@ static const int PERIOD_PATHMANAGER_MS = 100;
 static const int PERIOD_TELEMETRY_MS = 100; 
 static const int PERIOD_SENSORFUSION_MS = 200; 
 static const int PERIOD_INTERCHIP_MS = 20;
+static const int PERIOD_IMU_MS = 5; // Needs to be called at 200 Hz. 
 
-static bool catastrophicFailure = false;
+static volatile bool catastrophicFailure = false;
```

Comment:
Aadi added this to his pr which was just merged so there will be merge conflicts... should not be too bad to solve tho looking at the affected files